File: PB - More Exercises JavaScript/6.0 Drawing Figures with Loops - More Exercises/06. Rhombus of Stars.js
function rhombusOfStars(n) {
    //first part
    for (let i = 1; i <= n; i++) {
        console.log(`${' '.repeat(n - i) }${'* '.repeat(i)}`)
    }

    //second part
    for (let i = 1; i < n; i++) {
        console.log(`${' '.repeat(i) }${'* '.repeat(n -i)}`)
    }
}

// rhombusOfStars(4)

File: PB - More Exercises GO/6.0 Drawing Figures with Loops - More Exercises/06. Rhombus of Stars.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var n int
	fmt.Scan(&n)

	//first part
	for i := 1; i < n; i++ {
		fmt.Println(strings.Repeat(" ", n-i-1), strings.Repeat("* ", i))

	}

	//middle
	fmt.Println(strings.Repeat("* ", n))

	//end part
	for i := 1; i < n; i++ {
		fmt.Println(strings.Repeat(" ", i-1), strings.Repeat("* ", n-i))
	}
}

/*
name   :06. Rhombus of Stars
input  :3
output :
   *
  * *
 * * *
  * *
   *
*/
